tests/protractor/page-objects/forms/delivery-report.po.js

```diff
@@ -1171,36 +1143,35 @@ module.exports = {
 
     element(by.css('.selection')).click();
     const search = element(by.css('.select2-search__field'));
+    search.click();
     search.sendKeys(name);
-    helper.waitElementToBeClickable(element(by.css('[role="treeitem"]')));
-    search.sendKeys(protractor.Key.ENTER);
-    element(by.css('[role="treeitem"]')).click();
+    helper.waitElementToBeVisisble(element(by.css('.name')));
+    element(by.css('.name')).click();
   },
 
   //Delivery Info page -- Pregnancy outcomes
   selectLiveBirthButton: () => {
-    element(by.css('[value="health"]')).click();
+    selectRadioButton('healthy');
```

Comment:
Not sure if there was a typo in the first code or if the underlying html has changed, but atm, the value is "healthy" ..`<input type="radio" name="/delivery/group_delivery_summary/g_pregnancy_outcome" data-name="/delivery/group_delivery_summary/g_pregnancy_outcome" **value="healthy"** data-required="true()" data-type-xml="select1" class="no-unselect">`